tests/consumers/duckdb_consumer.py

```diff
@@ -0,0 +1,64 @@
+import os
+import string
+from collections.abc import Iterable
+
+import pyarrow as pa
+
+
+class DuckDBConsumer:
+    """
+    Implementation of the DuckDB substrait consumer Class for testing
+    """
+
+    @staticmethod
+    def run_substrait_query(db_connection, sql_query: str) -> pa.Table:
+        """
+        Convert the SQL query into a substrait query plan and run the plan against DuckDB.
+
+        Parameters:
+            db_connection:
+                DuckDB connection.
+            sql_query:
+                A SQL query string.
+
+        Returns:
+            A pyarrow table resulting from running the substrait query plan.
+        """
+        duckdb_substrait_plan = db_connection.get_substrait(sql_query)
+        proto_bytes = duckdb_substrait_plan.fetchone()[0]
+        return db_connection.from_substrait(proto_bytes).arrow()
+
+    @staticmethod
+    def load_tables_from_parquet(
+        db_connection,
```

Comment:
Both of these methods need the `db_connection` -- it could be kept with the `DuckDBConsumer` class and shoved into an attribute so it isn't required to be passed to both static methods (which would no longer be static)

Reply:
Thanks for the suggestion, this is cleaner!